frontend/src/component/common/TooltipResolver/TooltipResolver.tsx

```diff
@@ -1,11 +1,11 @@
-import React, { ReactNode } from 'react';
+import { ReactNode } from 'react';
 import { Tooltip, TooltipProps } from '@mui/material';
 import { HtmlTooltip } from '../HtmlTooltip/HtmlTooltip';
 
 export interface ITooltipResolverProps extends Omit<TooltipProps, 'title'> {
-    title?: string | undefined;
-    titleComponent?: ReactNode | undefined;
-    variant?: 'default' | 'white';
+    title?: string;
+    titleComponent?: ReactNode;
+    variant?: 'default' | 'custom';
```

Comment:
I think this should be `custom` (or `html` or something else). `white` doesn't fully represent all the `HtmlTooltip` component differences and e.g. dark mode would not exactly make this a "white" tooltip.

Reply:
Makes sense